Python/Product/Analysis/LanguageServer/CompletionAnalysis.cs

```diff
@@ -245,10 +245,8 @@ Func<TextReader> openDocument
                 }
                 foreach (var t in ZipLongest(imp.Names, imp.AsNames).Reverse()) {
                     if (t.Item2 != null) {
-                        if (Index > t.Item2.EndIndex && t.Item2.EndIndex > t.Item2.StartIndex) {
+                        if (Index >= t.Item2.StartIndex && t.Item2.EndIndex > t.Item2.StartIndex) {
```

Comment:
Does it actually matter that `t.Item2.EndIndex > t.Item2.StartIndex`?

Reply:
Yes. We can get empty names in this part of the ast for partially typed statements.

Reply:
Oh wait, I see what you mean (had other changes here in my head). Yeah, this time it doesn't matter, but generally when we're comparing against `EndIndex` it does.